utilities/ggd/README.md

```diff
@@ -0,0 +1,48 @@
+ggd - GoGitDiff
+===============
+
+This is a tool to compute the packages affected by the difference between two Git
+revisions. It's primary goal is to help speed up CI jobs by figuring out the packages
+affected by git changes, and transitively looking through the Go Imports. It takes
+inspiration from `gta`, and the work done at [DigitalOcean].
+
+NB: debug mode (`-d`) allows users to inspect how the tool arrives at the decisions it does.
```

Comment:
"allows users to inspect the tool's decision making process"

Reply:
Fair.